### PR TITLE
update zh-Hant.lproj

### PR DESCRIPTION
**Pull request by Neo1102**
Modify the languageCode to comply with Apple's developer documentation.
translate missing string

### CODE REVIEW

**Comment by L1ghtmann** [comment]
Could you possibly link the relevant docs? This change was previously reverted by @olivertzeng (https://github.com/L1ghtmann/IAmLazy-Localization/pull/8)

File: zh-Hant.lproj/Localizable.strings [hunk 1]
```diff
@@ -110,4 +110,4 @@
 "%@ has no debs!" = "%@ 沒有 deb 檔案！";
 "Failed to install %@!" = "安裝 %@ 失敗！";
 "The backup you're trying to restore from was made for %@ jailbreaks." = "您嘗試還原的備份是由%@ 越獄而製作的。";
-"Your current jailbreak is %@!" = "你目前的越獄是 %@!";
+"Your current jailbreak is %@!" = "您目前的越獄是 %@!";
```
Comment (olivertzeng) [inline zh-Hant.lproj/Localizable.strings:113]: 這我是沒什麼問題
單純個人選擇我尊重（我是習慣「你」）

File: zh-Hant.lproj/Localizable.strings [hunk 2]
```diff
@@ -7,7 +7,7 @@
 "Restore" = "還原";
 "Credits" = "銘謝";
 "Backups" = "備份檔案";
-"Easily backup and restore your tweaks" = "Easily backup and restore your tweaks";
+"Easily backup and restore your tweaks" = "輕鬆備份和恢復您的插體";
```
Comment (olivertzeng) [inline zh-Hant.lproj/Localizable.strings:10]: 輕鬆備份和「回復」（恢復有點像是恢復元氣 應該不太能用在這裡 要改是 optional）您的「插件」（台灣月育使用者幾乎沒在使用「插體」之前在 Dopamine 上有說過）

File: zh-Hant.lproj/Localizable.strings [hunk 3]
```diff
@@ -23,7 +23,7 @@
 "Confirm" = "確認";
 "Cancel" = "取消";
 
-"Tweak backup completed successfully in %@ seconds!" = "外掛備份在 %@ 秒內成功完成！";
+"Tweak backup completed successfully in %@ seconds!" = "插體備份在 %@ 秒內成功完成！";
```
Comment (olivertzeng) [inline zh-Hant.lproj/Localizable.strings:26]: 「插件」

File: zh-Hant.lproj/Localizable.strings [hunk 4]
```diff
@@ -54,25 +54,25 @@
 "Info: %@" = "詳細資訊：%@";
 "%@ cannot be deleted?!" = "%@ 無法刪除？！";
 "An error occured and %@ could not be imported!" = "發生錯誤，無法導入 %@！";
-"Swipe or tap desired backup" = "Swipe or tap desired backup";
-"Delete" = "Delete";
+"Swipe or tap desired backup" = "滑動或點擊所需的備份";
+"Delete" = "刪除";
 "Type: %@" = "Type: %@";
```
Comment (olivertzeng) [inline zh-Hant.lproj/Localizable.strings:59]: 「備份類型：%@」

File: zh-Hant.lproj/Localizable.strings [hunk 5]
```diff
@@ -54,25 +54,25 @@
...
-"Turkish translation" = "Turkish translation";
-"Japanese translation" = "Japanese translation";
-"Russian translation" = "Russian translation";
-"Arabic translation" = "Arabic translation";
-"Spanish translation" = "Spanish translation";
-"Simplified Chinese translation" = "Simplified Chinese translation";
-"Taiwanese Mandarin translation" = "Taiwanese Mandarin translation";
+"Thank you to all who have helped!" = "感謝所有提供過幫助的人！";
+"Reachability project" = "專案可行性";
+"Design inspiration (re: Electra)" = "設計靈感 (re: Electra)";
```
Comment (olivertzeng) [inline zh-Hant.lproj/Localizable.strings:64]: 「感謝 Electra 越獄提供 GUI 設計靈感」

File: zh-Hant.lproj/Localizable.strings [hunk 6]
```diff
@@ -54,25 +54,25 @@
...
+"Bootstrap package identification" = "Bootstrap套件辨識";
+"Archival functions" = "歸檔功能";
```
Comment (olivertzeng) [inline zh-Hant.lproj/Localizable.strings:66]: 「壓縮」功能

**Comment by olivertzeng** [comment]
@Neo1102 @L1ghtmann I've done my reviews on this project. If you have any questions you can ping me here, I'm willing to help.

**Comment by olivertzeng** [comment]
> Could you possibly link the relevant docs? This change was previously reverted by @olivertzeng (#8)

Again, I prefer zh_TW over zh_Hant but it's personal prefs so it doesn't really matter anyways

**Comment by olivertzeng** [comment]
Thanks for the translations that I've missed @Neo1102

**Review by olivertzeng — CHANGES_REQUESTED** [review]
台灣國語是台灣在地的「口音 accent」，不是一種語言
一般我都會稱之為「繁體中文」
或是「繁體中文（台灣）」
還有目前我在台灣論壇內沒有看過有人說插體
請修改成「插件」（或是請提供任何台灣論壇有說「插體」的
image
image
請修改

**Comment by Neo1102** [comment]
@L1ghtmann 
Here~
https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPInternational/LanguageandLocaleIDs/LanguageandLocaleIDs.html

I have now found a Swift document, which is written as zh_TW. 
https://developer.apple.com/documentation/sirikit/registering_custom_vocabulary_with_sirikit/global_vocabulary_reference/localizing_your_vocabulary_for_chinese_dialects/

My guess is that zh-Hant is probably the naming convention during the Objective-C era. There is more than one country using Traditional Chinese, and Swift directly uses the country/region as the language code to solve this problem.
Please decide which naming convention to use, I will follow your decision.

**Comment by Neo1102** [comment]
@olivertzeng 
插體這種說法來自皮樂
詞語結構就跟軟體 硬體 韌體一樣

這種附加性質的東西 目前並沒有正式的名稱
有看過 附加元件 擴充功能 增強套件 外掛 等等
插件這種講法 很明顯是支語
拿去Google搜尋 出來的結果更多是中國網站
只是跟素質一樣 從兩千零幾年就有人在講了
很多人不覺得或不知道他是而已

**Comment by olivertzeng** [comment]
> @olivertzeng 插體這種說法來自皮樂 詞語結構就跟軟體 硬體 韌體一樣
> 
> 這種附加性質的東西 目前並沒有正式的名稱 有看過 附加元件 擴充功能 增強套件 外掛 等等 插件這種講法 很明顯是支語 拿去Google搜尋 出來的結果更多是中國網站 只是跟素質一樣 從兩千零幾年就有人在講了 很多人不覺得或不知道他是而已

https://www.facebook.com/hiraku.tw/posts/365859498231862/
查了一下就找到了
可是我提出幾點異議
1. 那是他自己個人的意見，並不代表權台灣人
2. 即使「插件」這個詞是從中國傳來的，台灣人也早已習慣了，反而講「插體」會聽不懂你在說什麼
不如用「套件」來詮釋 Tweak 好了 因為ios 也是用apt 而apt的packages也叫做套件
你覺得呢？
如果你同意我dopmaine 2的翻譯pr 一併修改

**Comment by Neo1102** [comment]
@olivertzeng 
要翻套件 我會把前面兩個字補上
如果要字少一點
我偏好翻 外掛 

至於你說的台灣國語
如果是要表達那種咬字不清的口音
我會直接翻 台灣狗以
要政治正確一點可以改成中華民國國語
繁體中文不止一國在用 現在我不偏好這種翻法
若要遵循Swift的語言碼 跟作者的原文
可以翻 台灣官方語 或直接翻 台灣中文

dopmaine的事就留到那個PR底下再談
這邊就不多說了

**Comment by olivertzeng** [comment]
外掛也可以
 好啊 可以改成台灣中文

File: zh-Hant.lproj/Localizable.strings (hunk 1 above)
Comment (Neo1102) [reply zh-Hant.lproj/Localizable.strings:113]: 軟體上大多使用尊稱
尤其是商業軟體

**Review by olivertzeng — APPROVED** [review]

**Comment by olivertzeng** [comment]
@L1ghtmann I've approved the PR

**Comment by L1ghtmann** [comment]
Thank you both for your assistance! 

> Generally I will call it "Chinese Traditional" or "Chinese Traditional (Taiwan)"

Noted. I will change the key (English version) to "Traditional Chinese" in an update.

**Comment by L1ghtmann** [comment]
@Neo1102 are you finished with your edits?

**Comment by olivertzeng** [comment]
> @Neo1102 are you finished with your edits?

he is

**Comment by Neo1102** [comment]
> @L1ghtmann Here~ https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPInternational/LanguageandLocaleIDs/LanguageandLocaleIDs.html
> 
> I have now found a Swift document, which is written as zh_TW. https://developer.apple.com/documentation/sirikit/registering_custom_vocabulary_with_sirikit/global_vocabulary_reference/localizing_your_vocabulary_for_chinese_dialects/
> 
> My guess is that zh-Hant is probably the naming convention during the Objective-C era. There is more than one country using Traditional Chinese, and Swift directly uses the country/region as the language code to solve this problem. Please decide which naming convention to use, I will follow your decision.

Regarding language code, which naming convention have you decided to use?

**Comment by L1ghtmann** [comment]
-Hant seems to be more generic, which is fitting given that, as you mentioned, tranditional Chinese is used in places other than just Taiwan.

**Comment by Neo1102** [comment]
However, in places that also use Traditional Chinese (such as Hong Kong), there are differences in terminology due to cultural differences. If zh-TW and zh-HK are used, Hong Kong users can maintain their own localized translations. If Hant is used, it will be mixed together.

**Comment by L1ghtmann** [comment]
Hmmm, taking a look at this again I'm a bit torn. What would you two recommend given the things mentioned? Keep as -TW so that there can be region-specific localizations or generalize?

**Comment by Neo1102** [comment]
Yes

**Review by L1ghtmann — APPROVED** [review]
Sounds good. Thanks again to you both!

**Pull request merged**